installers/conda/win/create_package.sh

```diff
@@ -0,0 +1,226 @@
+#!/bin/bash -e
+
+# Constructs a standalone windows MantidWorkbench installer using NSIS.
+# The package is created from a pre-packaged version of Mantid from conda
+# and removes any excess that is not necessary in a standalone
+# package
+
+# Print usage and exit
+function usage() {
+  local exitcode=$1
+  echo "Usage: $0 [options] package_name"
+  echo
+  echo "Create a standalone installable package out of a mantidworkbench Conda package. The"
+  echo " package is built in $BUILDDIR. This directory will be created if it does not exist"
+  echo " or purged if it already exists. The final installer will be created in the current"
+  echo " working directory. Requires mamba and NSIS to be installed in the running "
+  echo "environment, and on the path."
```

Comment:
Is this still correct? I know nsis is in the running environment but does it need to be on the path?

Reply:
Good point, NSIS is no longer required at all for this to run because we install it on the fly. I've updated the output so that it's correct now.